mpitest/test_mpi_derivs.py

```diff
@@ -22,37 +21,43 @@
 try:
     from mpi4py import MPI
     from openmdao.core.petsc_impl import PetscImpl as impl
+    from openmdao.solvers.petsc_ksp import PetscKSP
 except ImportError:
     impl = None
 
-class TestPetscKSP(MPITestCase):
+class FanOutGrouped(Group):
```

Comment:
This class is  specific to what I was testing here. It's not really intended to be reused elsewhere.